packages/contracts/test/plugins/governance/multisig/multisig-constants.ts

```diff
@@ -0,0 +1,22 @@
+import {ethers} from 'hardhat';
+
+export const MULTISIG_EVENTS = {
+  MULTISIG_SETTINGS_UPDATED: 'MultisigSettingsUpdated',
+  APPROVED: 'Approved',
+};
+
+export const MULTISIG_INTERFACE = new ethers.utils.Interface([
+  'function initialize(address,address[],tuple(bool,uint16))',
+  'function updateMultisigSettings(tuple(bool,uint16))',
+  'function createProposal(bytes,tuple(address,uint256,bytes)[],uint256,bool,bool,uint64,uint64) ',
+  'function getProposal(uint256)',
+]);
+
+export const UPDATE_MULTISIG_SETTINGS_PERMISSION_ID = ethers.utils.id(
```

Comment:
same question as within the admin plugin. Do we follow the pattern exported by the commons-sdk?

Reply:
These permission IDs should go into the separate plugin repos after we've split them.